Doc/library/logging.rst

```diff
@@ -242,6 +242,10 @@ is the module's name in the Python package namespace.
       above example). In such circumstances, it is likely that specialized
       :class:`Formatter`\ s would be used with particular :class:`Handler`\ s.
 
+      If no handler is attached to this logger (or its parents if
```

Comment:
Perhaps change to
```
If no handler is attached to this logger (or any of its ancestors,
taking into account the relevant :attr:`Logger.propagate` attributes),
the message will be sent ...
```

Reply:
Addressed on d173e5b75404a6dbf8883df24928f53795cef2ad